### PR TITLE
security doc: update allow cert config

### PR DESCRIPTION
**Pull request by hunterlxt**
Signed-off-by: TXXT <hunterlxt@live.com>

### What is changed?

update allow cert config

### Any related PRs or issues?

https://github.com/tikv/tikv/pull/7134
https://github.com/tikv/tikv/commit/e1b456e3debee40fbbbc38ebdc3ed0ddae9f3b6d

### Which version does your change affect?

TiKV master, release 4.0

### CODE REVIEW

**Comment by hunterlxt** [comment]
> LGTM. Do we need to update the 3.0 version?

@dcalvin Yes, just 3.1

**Comment by hunterlxt** [comment]
@zhangjinpeng1987 PTAL

File: content/docs/dev/tasks/configure/security.md [hunk 1]
```diff
@@ -101,6 +101,9 @@ ca-path = "/path/to/ca.pem"
 cert-path = "/path/to/tikv-server-cert.pem"
 # The path to the file that contains the PEM encoding of the server’s private key.
 key-path = "/path/to/tikv-server-key.pem"
+# The name list used to verify the common name in client’s certificates. Verification is
+# not enabled if this field is empty.
```
Comment (zhangjinpeng1987) [inline content/docs/dev/tasks/configure/security.md:105]: Can you give some examples of the names?

Comment (hunterlxt) [reply content/docs/dev/tasks/configure/security.md:105]: Done

**Comment by dcalvin** [comment]
> > LGTM. Do we need to update the 3.0 version?
> 
> @dcalvin Yes, just 3.1

As confirmed with @zhangjinpeng1987, we will not expose 3.1 on the website. Just dev (master). And when 4.0 GA is out, we will add 4.0 version.

**Comment by hunterlxt** [comment]
> > > LGTM. Do we need to update the 3.0 version?
> > 
> > 
> > @dcalvin Yes, just 3.1
> 
> As confirmed with @zhangjinpeng1987, we will not expose 3.1 on the website. Just dev (master). And when 4.0 GA is out, we will add 4.0 version.

@dcalvin Removed in PR description.
@zhangjinpeng1987 PTAL

**Pull request merged**